### PR TITLE
Add deploy to EC2 on merge or push into development

### PR DESCRIPTION
**Pull request by morrieinmaas**
NOTE: I currently dont have maintainer rights in the repo so I cannot
set up the secrets required for this to run successfully

### CODE REVIEW

**Comment by morrieinmaas** [comment]
Alright - ready for testing this out  - I added the secrets

**Review by TimoGlastra — APPROVED** [review]
Nice and simple! :)

File: .github/workflows/continuous-build.yml [hunk 1]
```diff
@@ -64,5 +64,20 @@ jobs:
          -Dsonar.python.pylint.reportPaths=pylintreport.txt
          -Dsonar.projectBaseDir=./app
          -Dsonar.coverage.exclusions=tests/**/*
-
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
```
Comment (TimoGlastra) [inline .github/workflows/continuous-build.yml:69]: maybe add a `requires: [build]`?

Comment (morrieinmaas) [reply .github/workflows/continuous-build.yml:69]: good shout let me have a go at that

Comment (morrieinmaas) [reply .github/workflows/continuous-build.yml:70]: I used `needs:` - [see docs](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idneeds)

**Pull request merged**